moodlecli/main.py

```diff
@@ -250,3 +250,33 @@ def export_users_by_course(source_course_id, bucket_name, key):
     )
 
     aws.put_json_data(users, bucket_name, key)
+
+
+@cli.command()
+@click.argument('bucket_name')
+@click.argument('directory')
+def export_users_bulk(bucket_name, directory):
```

Comment:
It noticed this in this PR, but currently we have:

```
export-users-bulk
export-grades-bulk
export-users-by-course
export-grades
```

Should we make `export-users-by-course` just `export-grades` so they're symmetric?

Reply:
Yea, I think that'd be advisable